projects/components/src/search-box/search-box.component.test.ts

```diff
@@ -13,32 +12,29 @@ describe('Search box Component', () => {
   });
 
   test('should work with default values', fakeAsync(() => {
-    spectator = createHost(
-      `<ht-search-box [placeholder]="placeholder" (valueChange)="onValueChange($event)"></ht-search-box>`,
-      {
-        hostProps: {
-          placeholder: 'Test Placeholder'
-        }
+    spectator = createHost(`<ht-search-box [placeholder]="placeholder"></ht-search-box>`, {
```

Comment:
This test was logging errors. Updated to use spectator and remove the undeclared listener.